src/templates/video_player.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layout.html" %}
+
+{% block body %}
+<section class="content">
+  <div class="container">
+    <a href="/">
+      <img class="logo" src="/static/images/logo.svg" alt="Emmental Logo">
+    </a>
+    <div class="user-buttons">
+      <a href="/">
+        <button class="logout-button" type="button">Manage</button>
+      </a>
+      <a href="/logout">
+        <button class="logout-button" type="button">Logout</button>
+      </a>
+    </div>
+    <!-- needs a button for main page-->
+
+    {% with messages = get_flashed_messages() %}
+    {% if messages %}
+    <div class="flash-messages">
+      {% for message in messages %}
+      <p>{{ message|safe }}</p>
+      {% endfor %}
+    </div>
+    {% endif %}
+    {% endwith %}
+
+    <h1>Video Player:</h1>    
+    {% if video %}
+    <div class="videos">
+        <video width="75%" controls>
+          <source src="/video/{{ video }}" type="video/mp4">
```

Comment:
Change type to `type="video/{{ video.extension }}"`

Reply:
Yeah. I only need to path to the video link. So, that's all I pass.